data_steward/validation/main.py

```diff
@@ -696,8 +697,10 @@ def perform_validation_on_file(file_name, found_file_names, hpo_id,
 
 
 def _validation_done(bucket, folder):
-    if gcs_utils.get_metadata(bucket=bucket,
-                              name=folder + common.PROCESSED_TXT) is not None:
+    storage_client = StorageClient()
+    sc_bucket = storage_client.get_bucket(bucket)
+    bucket_blob = sc_bucket.blob(f'{folder}{common.PROCESSED_TXT}')
```

Comment:
This is fine here as well, but important to note that bucket.blob does not make an HTTP request ([see example here](https://cloud.google.com/storage/docs/viewing-editing-metadata#storage-view-object-metadata-python)). However, since it is only calling get_blob_metadata below (which has an added check to make an HTTP request if not already done), it is handled there. However, I'd suggest taking a step back and figuring out what is being achieved. It is looking for the creation of a specific file. Try writing a simple integration test to see if this really figures out if a blob exists or not. If not, what would be the best way to check if a blob exists or not via [their docs](https://googleapis.dev/python/storage/latest/blobs.html)?

Reply:
Yes. line 702 has to go because we are not creating a blob.

Reply:
Yes. We don't need the metadata because we just have to check that the blob exists.